src/util/customTransformer.js

```diff
@@ -78,6 +79,26 @@ async function runUserTransform(
     }),
   );
 
+  await jail.set(
+    '_getGeoLocation',
```

Comment:
Given the implementations are same here and in ivmFactory, should we think about refactoring it into a shared implementation somewhere ?

Reply:
we are deprecating this code.